torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4342,6 +4373,21 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            op=torch.fmin,
            dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
            sample_inputs_func=sample_inputs_max_min_binary,),
+    OpInfo('fmod',
+           dtypes=all_types_and(torch.float16, torch.bool),
+           sample_inputs_func=sample_inputs_fmod_remainder,
+           skips=(
+               SkipInfo('TestCommon', 'test_variant_consistency_jit'),
+               SkipInfo('TestCommon', 'test_variant_consistency_eager'),
```

Comment:
Could you elaborate on why the tests are failing? Probably add a small comment explaining the same.